README.md

```diff
@@ -60,6 +60,10 @@ Done predicting. Predict table : iris.predict
 - [The walkthrough of the source code](doc/walkthrough.md)
 - [The choice of parser generator](doc/sql_parser.md)
 
+## Roadmap
+
+Various data sources and ML frameworks support is a work in progress. Please refer to our [roadmap](https://github.com/sql-machine-learning/sqlflow/issues/327) for specific timelines. Also, let us know your interests so that we can prioritize our work.
```

Comment:
My personal suggestions for the content: 

SQLFlow will love to support as many mainstream ML frameworks and data sources as possible, but we feel like the expansion would be hard to be done merely on our own, so we would love to hear your options on what ML frameworks and data sources you are currently using and build upon. Please refer to our [roadmap](https://github.com/sql-machine-learning/sqlflow/issues/327) for specific timelines, also let us know your current scenarios and interests around SQLFlow project so we can prioritize based on the feedback from the community.